libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Util/SFSDKViewUtils.m

```diff
@@ -65,6 +64,9 @@ + (void)styleNavigationBar:(UINavigationBar *)navigationBar config:(SFSDKLoginVi
     }
     
     if ([textAttributes count] > 0) {
+        if (@available(iOS 13.0, *)) {
+            navigationBar.standardAppearance.titleTextAttributes = textAttributes;
```

Comment:
I don't know the root cause for this one -- the custom attributes weren’t showing on iOS 13 without this. It looks like the text on the login nav bar is correctly styled because additional styling is happening on the label in `createTitleItem`,  but the title on host picker is only styled by this method and wasn’t showing the correct attributes until I tried this